src/libraries/System.Private.CoreLib/src/System/Threading/WaitSubsystem.WaitableObject.Unix.cs

```diff
@@ -220,8 +220,8 @@ private void AcceptSignal(ThreadWaitInfo waitInfo)
                         Debug.Assert(_type == WaitableObjectType.Mutex);
                         --_signalCount;
 
-                        Debug.Assert(_ownershipInfo.Thread == null);
-                        _ownershipInfo.AssignOwnership(this, waitInfo);
+                        Debug.Assert(_ownershipInfo!.Thread == null);
+                        _ownershipInfo!.AssignOwnership(this, waitInfo);
```

Comment:
I'm surprised this `!` is required; I'd have thought the one on the previous line would have negated the need for this one.